deploy-board/deploy_board/static/js/deploy-board.js

```diff
@@ -202,60 +233,31 @@ function getDefaultPlacement(capacityCreationInfo) {
                     return {
                         value: item.id,
                         text: item.abstract_name,
-                        isSelected: this.isDefault(item)
+                        isSelected: inDefault(item)
                     }
                 })
             }
-            return arr;
-
+            return convertToPlacementOptions(arr);
         },
         getFullList: function (showPublicOnly, existingItems) {
```

Comment:
do we have any unit testing framework for js in deploy board?

Reply:
No, we don't.